.circleci/config.yml

```diff
@@ -19,7 +19,17 @@ jobs:
           paths:
             - node_modules
           key: v1-dependencies-{{ checksum "package.json" }}
-      # run tests!
+  test:
+    docker:
+      - image: circleci/node:lts-jessie
+
+    working_directory: ~/repo
+
+    steps:
+      - checkout
+      - run: 
+          name: Build and install dependencies
+          command: npm install
```

Comment:
use dependencies caching

Reply:
Yes you are right. We are using it on the build part but not in the test one. I will fix it.

Reply:
The cache is now restored into the test step and I prefer to save a new version of the cache only into the building one. As normally both two is downloading the same files no need to create cache twice.